units/user/systemd-tmpfiles-setup.service

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Create User's Volatile Files and Directories
+Description=Create User-wide Files and Directories
```

Comment:
"User-wide" is strange ;) Maybe just "Create User Files and Directories"? 

(Formally, "User's" is the correct grammatical form. Or "Users'". But everybody says "User files"  and "Kernel Config" and so on.)

Reply:
Actually, maybe we don't even need "User"? It's under the user manager, so the scope is implicit.